app/src/userpages/components/StreamPage/Show/HistoryView/index.jsx

```diff
@@ -316,6 +319,7 @@ class HistoryView extends Component<Props, State> {
                         </Row>
                     </Fragment>
                 )}
+                {stream && stream.storageDays &&
```

Comment:
Maybe check for `!== undefined` here so it works if `storageDays` is zero?

Reply:
ahh, really good point! Will fix.